### DISCUSSION TITLE
Help with Scaling an AWS instance for an event?

### DISCUSSION THREAD

**Discussion by jonoakes** [mozilla/hubs#3353] [Hubs Cloud Questions] [answered]
We are having an event which will host up to 400 users across 40 rooms for an hour or two.

Is this possible to achieve on a personal AWS account? What would be a good configuration for this situation? (on a budget!) :)

**Comment by robinkwilson** [comment, marked as answer]
Totally possible on a Personal, I'd suggest the c5.4xlarge *during* the event. From [the cost/CCU charts (alpha)](https://hubs.mozilla.com/docs/hubs-cloud-aws-estimated-cost-charts.html#estimating-personal--enterprise-costs-with-1-server).

Remember to review The [Minimizing costs user story](https://hubs.mozilla.com/docs/hubs-cloud-aws-costs.html#minimize-your-costs---a-user-story) to get a picture of how to decrease costs leading up to the event.
